lib/router/src/utils.ts

```diff
@@ -0,0 +1,29 @@
+import qs from 'qs';
+import memoize from 'memoizerific';
+
+interface StoryData {
+  viewMode?: string;
+  storyId?: string;
+}
+
+export const storyDataFromString: (path: string) => StoryData = memoize(1000)((path: string | undefined | null) => {
+  const result: StoryData = {
+    viewMode: undefined,
+    storyId: undefined,
+  };
+
+  if (path) {
+    const [, p1, p2] = path.match(/\/([^/]+)\/([^/]+)?/) || [undefined, undefined, undefined];
```

Comment:
Extract regexp to constant
```
const reg = /\/([^/]+)\/([^/]+)?/;
const [, p1 = undefined, p2 = undefined] = path.match(reg);
```

Reply:
And give proper names `viewMode` and `storyId`;

Reply:
✅